pwa/src/templates/organizationsTemplate/OrganizationsTemplate.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import * as styles from "./OrganizationsTemplate.module.css";
+import { Container } from "@conduction/components";
+import { FiltersContext } from "../../context/filters";
+import { useTranslation } from "react-i18next";
+import { QueryClient } from "react-query";
+import Skeleton from "react-loading-skeleton";
+import { GatsbyContext } from "../../context/gatsby";
+import { PaginatedItems } from "../../components/pagination/pagination";
+import ResultsDisplaySwitch from "../../components/resultsDisplaySwitch/ResultsDisplaySwitch";
+import { Heading } from "@utrecht/component-library-react/dist/css-module";
+import { useOrganization } from "../../hooks/organization";
+import { OrganizationHorizontalFiltersTemplate } from "../templateParts/filters/horizontalFilters/HorizontalFiltersTemplate";
+import { OrganizationDisplayTemplate } from "../templateParts/OrganizationDisplayTemplates/OrganizationDisplayTemplate";
+
+export const OrganizationsTemplate: React.FC = (props) => {
+  const [filters, setFilters] = React.useContext(FiltersContext);
+  const { t } = useTranslation();
+  const { screenSize } = React.useContext(GatsbyContext);
+  const [marginPagesDisplayed, setMarginPageDisplayed] = React.useState<number>(3);
+
+  const queryClient = new QueryClient();
+  const _useOrganisation = useOrganization(queryClient);
+  const getOrganisations = _useOrganisation.getAll({ ...filters, organizationsResultDisplayLayout: "cards" });
+
+  React.useEffect(() => {
+    if (getOrganisations.isSuccess && screenSize === "mobile") {
+      setMarginPageDisplayed(2);
+    }
+    if (getOrganisations.isSuccess && screenSize === "mobile" && getOrganisations.data.pages > 100) {
+      setMarginPageDisplayed(1);
+    }
+    if (getOrganisations.isSuccess && screenSize !== "mobile") {
+      setMarginPageDisplayed(3);
+    }
+  }, [getOrganisations]);
```

Comment:
I'm thinking we might not need this in the implementation layer, couldn't `PaginatedItems` handle this within its own context? 

It knows how many pages it's showing.